WalletWasabi.Fluent/ViewModels/Wallets/Home/History/HistoryViewModel.cs

```diff
@@ -229,7 +228,8 @@ private async Task UpdateAsync()
 		{
 			var historyBuilder = new TransactionHistoryBuilder(_walletViewModel.Wallet);
 			var rawHistoryList = await Task.Run(historyBuilder.BuildHistorySummary);
-			var newHistoryList = GenerateHistoryList(rawHistoryList).ToArray();
+			var orderedRawHistoryList = rawHistoryList.OrderBy(x => x.DateTime).ThenBy(x => x.Height).ToList();
```

Comment:
Please add ThenBy blockindex. DateTime and Height can easily be the same, and we cannot let it be random (I know we order it before so it won't be random, but it is not safe to assume that).